### PR TITLE
revamp(ui):  whats new alert box popup 

### PR DESCRIPTION
**Pull request by Ashish8689**
Issue https://github.com/open-metadata/OpenMetadata/issues/13055

### Describe your changes:

- I worked on revamping whats new alert box popup 
- Fix the typo error in the activity feed

image

https://github.com/open-metadata/OpenMetadata/assets/66266464/5e42b846-1b5c-45bb-9314-4b1b9f66933d

#
### Type of change:

- [ ] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/assets/img/update.gif
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/assets/img/update.gif:1]: can we name it better? update.gif does not self explanatory. also can we create separate folder for gif and move this gif there?

File: openmetadata-ui/src/main/resources/ui/src/assets/svg/ic-share.svg [hunk 1]
```diff
@@ -1,6 +1,6 @@
-<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/assets/svg/ic-share.svg:1]: as we are removing width and height, please verify the places where this svg is used. and if needed provide W/H from there.

File: openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/WhatsNewModal.styles.less [hunk 2]
```diff
@@ -37,27 +37,101 @@
...
+
 .whats-new-alert-container {
   position: fixed;
-  bottom: 24px;
+  bottom: -100%;
   right: 30px;
   z-index: 1;
+  background: @white;
+  animation: scrollIn 1s normal forwards ease-in-out;
+  animation-delay: 2s;
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/WhatsNewModal.styles.less:53]: i think we should use 1s, @chirag-madlani whats your thought? I am feeling its very delay.

File: openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/WhatsNewModal.styles.less [hunk 3]
```diff
@@ -37,27 +37,101 @@
...
     font-weight: 600;
     font-size: 16px;
   }
-  .whats-new-alert-description {
-    color: @white;
+
+  .whats-new-alert-content {
+    .whats-new-alert-sub-header,
+    .show-more,
+    a[href]:not(.button-comp):not(.no-underline):not(.link-text-info),
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/WhatsNewModal.styles.less:79]: dont we have any better way apart from styling like this?

File: openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/WhatsNewModal.styles.less [hunk 4]
```diff
@@ -37,27 +37,101 @@
...
+    .whats-new-alert-content-icon-arrow {
+      display: none;
+      width: 16px;
+      height: 16px;
+      color: @text-grey-muted;
+    }
+
+    &:hover {
+      .whats-new-alert-sub-header,
+      a[href]:not(.button-comp):not(.no-underline):not(.link-text-info),
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/WhatsNewModal.styles.less:107]: same here

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 5]
```diff
@@ -689,6 +689,7 @@
     "open-lowercase": "open",
     "open-metadata": "OpenMetadata",
     "open-metadata-logo": "OpenMetadata Logo",
+    "open-metadata-updated": "OpenMetadata Updated !",
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:692]:
```suggestion
    "open-metadata-updated": "OpenMetadata Updated!",
```
I think space is not necessary here, @ShilpaVernekar any thoughts?

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 6]
```diff
@@ -1132,6 +1134,7 @@
     "wednesday": "Wednesday",
     "week": "Week",
     "weekly-usage": "Weekly Usage",
+    "what-new-version": " What's New ({{version}})",
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:1137]:
```suggestion
    "whats-new-version": " What's New ({{version}})",
```

**Review by ShaileshParmar11 — CHANGES_REQUESTED** [review]
This changes will impact all the cypress test case, can you please verify and update it.

File: openmetadata-ui/src/main/resources/ui/src/assets/svg/ic-share.svg (hunk 1 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/assets/svg/ic-share.svg:1]: yes, it is verified and none is breaking!

**Comment by Ashish8689** [comment]
@ShaileshParmar11  comments addressed!

**Review by ShaileshParmar11 — APPROVED** [review]

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**